package.json

```diff
@@ -27,23 +27,23 @@
     "statistics"
   ],
   "dependencies": {
-    "bar-horizontal": "^0.2.0",
-    "charm": "^1.0.0",
-    "cli-color": "^1.0.0",
-    "configstore": "^1.0.0",
-    "fast-csv": "^0.6.0",
-    "fast-stats": "0.0.2",
-    "file-exists": "^0.1.1",
-    "gh-gist-owner": "^1.0.1",
-    "jsonfile": "^2.0.1",
+    "bar-horizontal": "^0.3.0",
+    "charm": "^1.0.2",
+    "cli-color": "^1.2.0",
+    "configstore": "^3.1.1",
+    "fast-csv": "^2.4.1",
+    "fast-stats": "0.0.3",
+    "file-exists": "^5.0.0",
+    "gh-gist-owner": "^1.1.0",
+    "jsonfile": "^4.0.0",
     "keypress": "^0.2.1",
     "lodash.max": "^4.0.1",
     "lodash.min": "^4.0.1",
     "math-avg": "^1.0.0",
-    "meow": "^3.1.0",
+    "meow": "^3.7.0",
     "mkdirp": "^0.5.1",
-    "pretty-ms": "^1.2.0",
-    "read": "^1.0.6",
+    "pretty-ms": "^3.0.1",
+    "read": "^1.0.7",
```

Comment:
Whoa whoa. This is too much of a change! Bumping major versions is a gamble, and I don't want to do it without thorough testing. So, let's keep this out of the scope of this particular PR. As I mentioned in the other issue's comment, I will freeze versions after merging this PR, test the whole module and if everything works, make a release.

Deprecated warnings suck, but not as much as a module that stops working!